.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,59 @@
+## Summary of Bug or Feature
+
+<!--
+Summarize the issue you are experiencing (or the feature you want to see added). Tell us what you were trying to do and what happened instead. Remember, this is _not_ a place to ask questions. For that, go to http://discuss.appium.io!
+
+THIS REPO IS FOR ISSUES RELATED TO THE PACKAGES IN appium/appium-ios ONLY!  See https://github.com/appium/appium/issues for general Appium issues.
+-->
+
+## Package(s) Affected (Required)
+
+<!--
+Check the package(s) this issue involves, and fill in the version number.
+-->
+
+- [ ] `@appium/idb` - version: <!-- version -->
+- [ ] `@appium/ios-device` - version: <!-- version -->
+- [ ] `@appium/ios-simulator` - version: <!-- version  -->
+- [ ] `@appium/ios-test-app` - version: <!-- version -->
+- [ ] `@appium/remote-debugger` - version: <!-- version -->
+- [ ] `@appium/xcode` - version: <!-- version -->
+
+## Details (Optional - Good to Have)
+
+<!--
+If possible, describe the problem you have been experiencing in more detail, add notes, etc. If this is a feature request: what might be an alternative solution?  Is there a workaround?
```

Comment:
Usually the format:

- steps to reproduce
- actual result
- expected result

works the best